Project.toml

```diff
@@ -7,14 +7,18 @@ version = "1.0.0"
 [deps]
 Dates = "ade2ca70-3891-5945-98fb-dc099432e06a"
 Printf = "de0858da-6303-5e67-8744-51eddeeeb8d7"
+RecipesBase = "3cdcf5f2-1ef4-517c-9805-6587b60abb01"
 TimeZones = "f269a46b-ccf7-5d73-abea-4c690281aa53"
 
 [compat]
 TimeZones = "0.7, 0.8, 0.9, 0.10, 0.11, 1"
```

Comment:
Do you need the plotting support for TimeZones?

Reply:
No we don't test that in the automated tests.

Reply:
Sorry, I wasn't clear enough. If someone was to plot a `HourEnding{ZonedDateTime}` does that work correctly without having plotting support for TimeZones.jl?

Reply:
Yes, that will word correctly. There is a demo of it in the TimeZones PR